megamek/src/megamek/server/victory/KillCountVictory.java

```diff
@@ -100,6 +102,19 @@ private void updateKillTables(IGame game,
             Entity killer = game.getEntityFromAllSources(wreck.getKillerId());
             
             if (killer == null){
+                if (game.getNoOfTeams() == 2 && wreck.getOwner().getTeam() != Player.TEAM_NONE) {
```

Comment:
Each of the comparisons should be surrounded with parents, aka `(game.getNoOfTeams() == 2) && (wreck.getOwner().getTeam() != Player.TEAM_NONE)`